pkg/flags/flags.go

```diff
@@ -33,6 +33,15 @@ func ConfigureAndParse(cmd *flag.FlagSet, args []string) {
 	maybePrintVersionAndExit(*printVersion)
 }
 
+// AddTraceFlags adds the trace-collector and sampling-probability flags
+// to the flagSet and returns their pointers for usage
+func AddTraceFlags(cmd *flag.FlagSet) (*string, *float64) {
+	traceCollector := cmd.String("trace-collector", "", "Enables OC Tracing with the specified endpoint as collector")
+	probabilitySampling := cmd.Float64("sampling-probability", 0, "The probabilistic sampling rate for the traces")
```

Comment:
Let's not add the ability to change sampling for the control plane. We can add it later and I suspect we'll want it at 100% most of the time.

Reply:
yeah, this will also allow us to have lesser configuration.

Reply:
@adleong is there a reason you wanted the sampling here? Feels like extra, unneeded configuration that we'd have to support.

Reply:
I'd be kinda tempted to leave this flag in here but not expose it through the install flag, just in case?  But I'm also fine taking it out altogether.

Reply:
I can be talked into just hiding it, worried it'll become stale after awhile though. @Pothulapati it is up to you, my preference would be removing it entirely, but if that's a bunch of work you could just hide the flag.

Reply:
Removed it. :p

Reply:
Sampling Config has also been removed in #3539